From a9c0ef3b8eed9558043d911115affa237df98d7a Mon Sep 17 00:00:00 2001
From: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Date: Wed, 23 Aug 2017 17:51:23 +0300
Subject: [PATCH 210/448] staging: fsl-dpaa2/mac: Comply with mc bus header updates

The fsl-mc bus driver reorganized some of its headers, so
update our includes accordingly.

Signed-off-by: Ioana Radulescu <ruxandra.radulescu@nxp.com>
---
 drivers/staging/fsl-dpaa2/mac/dpmac.c |    3 +--
 drivers/staging/fsl-dpaa2/mac/mac.c   |    1 -
 2 files changed, 1 insertions(+), 3 deletions(-)

diff --git a/drivers/staging/fsl-dpaa2/mac/dpmac.c b/drivers/staging/fsl-dpaa2/mac/dpmac.c
index f782742..baef769 100644
--- a/drivers/staging/fsl-dpaa2/mac/dpmac.c
+++ b/drivers/staging/fsl-dpaa2/mac/dpmac.c
@@ -29,8 +29,7 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
-#include "../../fsl-mc/include/mc-sys.h"
-#include "../../fsl-mc/include/mc-cmd.h"
+#include "../../fsl-mc/include/mc.h"
 #include "dpmac.h"
 #include "dpmac-cmd.h"
 
diff --git a/drivers/staging/fsl-dpaa2/mac/mac.c b/drivers/staging/fsl-dpaa2/mac/mac.c
index ee28c3a..e9c1af8 100644
--- a/drivers/staging/fsl-dpaa2/mac/mac.c
+++ b/drivers/staging/fsl-dpaa2/mac/mac.c
@@ -47,7 +47,6 @@
 #include <linux/phy_fixed.h>
 
 #include "../../fsl-mc/include/mc.h"
-#include "../../fsl-mc/include/mc-sys.h"
 
 #include "dpmac.h"
 #include "dpmac-cmd.h"
-- 
1.7.1

